### DISCUSSION TITLE
Import/Export Lutris Library

### DISCUSSION THREAD

**Discussion by trymeouteh** [lutris/lutris#4157] [Ideas]
Please add a setting to import and export your Lutris library into a compressed zip file which can be imported again. This compressed file will contain all of your used runners and games and its data such as the banner image, game info, game options, runner options, all of it.

This way Lutris will have an official import/export feature to allow for gamers to export their library and back it up now and then and then be able to import it to their new gaming computers with just a few clicks.

This is to import/export the Lutris library, not the games themselves but the data to be able to reconstruct the Lutris library on another computer.

**Comment by TERMINATOR461997** [comment]
I know this is extremely late. But for anyone asking a similar question I have a possible solution as long as you're willing to change for system preferences a bit.

1. Get a separate place to store data, I prefer an SD Card, its cheap, portable, and only needs to store my User libraries (Downloads, documents, etc). As long as its not the same drive whatever will do.
2. Create symlinks for your directories on the other drive. Anything from the 'Games' folder [A] or just the places where all the games will save to, appdata/XYZ, etc..
3. Assuming you're running a flatpak build, locate '.var/app/net.lutris.Lutris' at this point you can either symlink it as well, or just backup the location every so often. [B]

A. In this case drive speed will be a factor.
B. This was done on a Linux Mint 22 system. I don't believe the folder is in the same location for every distro, so you may need to hunt down the flatpak apps directory on yours. I'd start by looking in '.local/share/flatpak'
